### PR TITLE
Tiny js world

### PR DESCRIPTION
**Pull request by akulahere**
### Building a Tiny JS World

[Demo](https://akulahere.github.io/a-tiny-JS-world/)
[Code base](https://github.com/akulahere/a-tiny-JS-world)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/akulahere/a-tiny-js-world/index.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  name: 'Lero',
+  gender: 'female',
+  legs: 2,
+  hands: 2,
+  saying: 'Bye!'
+};
+
+const inhabitantsArray = [dog, cat, man, woman];
+
+const message = (elem) => `Species: ${elem.species}. Name: ${elem.name}. Gender: ${elem.gender}. ${elem.legs} legs and ${elem.hands} hands. Phrase: ${elem.saying}`;
```
Comment (OleksiyRudenko) [inline submissions/akulahere/a-tiny-js-world/index.js:36]: `elem` is to generic naming. We are dealing with inhabitants here. Also avoid abbreviations in naming.

File: submissions/akulahere/a-tiny-js-world/index.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+inhabitantsArray.forEach( inh => print(message(inh)) );
```
Comment (OleksiyRudenko) [inline submissions/akulahere/a-tiny-js-world/index.js:38]: Avoid abbreviations in naming. Take care of your code readers.

File: submissions/akulahere/a-tiny-js-world/index.js (hunk 1 above)
Comment (OleksiyRudenko) [inline submissions/akulahere/a-tiny-js-world/index.js:36]: Can you reduce code repetition here using `Array` methods?

Hints:
1. Converting property names into property values: ` ['propNameA', 'propNameB'].map(propName => obj[propName])`
2. [Capitalization](https://flaviocopes.com/how-to-uppercase-first-letter-javascript/)

**Review by OleksiyRudenko — CHANGES_REQUESTED** [review]
@akulahere good job.
Let's practise good naming and `Array` methods.

File: submissions/akulahere/a-tiny-js-world/index.js (hunk 1 above)
Comment (akulahere) [reply submissions/akulahere/a-tiny-js-world/index.js]: Refactored this part of code

Comment (akulahere) [reply submissions/akulahere/a-tiny-js-world/index.js]: Renamed

File: submissions/akulahere/a-tiny-js-world/index.js (hunk 2 above)
Comment (akulahere) [reply submissions/akulahere/a-tiny-js-world/index.js]: Renamed

**Review by OleksiyRudenko — APPROVED** [review]
@akulahere well done!

**Pull request merged**